src/tokenizer.scss

```diff
@@ -10,18 +10,24 @@ $block: #{$fd-namespace}-tokenizer;
 
   width: 100%;
   background-color: var(--sapField_Background);
-  padding-left: $fd-tokenizer-spacing;
-  overflow: hidden;
   display: flex;
   align-items: center;
 
   &__inner {
+    &::-webkit-scrollbar {
+      display: none; // hide scrollbar but keep element scrollable for Chrome/Safari/Opera
+    }
+
+    padding: 0 $fd-tokenizer-spacing;
+    overflow: hidden;
```

Comment:
The interaction spec states that the cozy (phone and tablet) tokenizer should be scrollable by swiping the tokens left and right.  So I think here we do need overflow-x: scroll